LiferayKotlinCodeFormatting.xml

```diff
@@ -0,0 +1,420 @@
+<code_scheme name="SquareAndroid" version="173">
```

Comment:
Both files are code schemes, right? One should the project convention, we have the same error in apio consumer that I fixed but I haven't uploaded the good project convention

Reply:
That's a very good question, we haven't defined one yet for kotlin projects. I think we can do it next week with Victor and I'll send it to you.